src/invocation/ConnectionAuthenticator.ts

```diff
@@ -43,7 +43,6 @@ export class ConnectionAuthenticator {
             .then((msg: ClientMessage) => {
                 var authResponse = ClientAuthenticationCodec.decodeResponse(msg);
                 if (authResponse.status === 0) {
-                    this.connection.address = authResponse.address;
```

Comment:
The address coming from server needs to be set to connection and connection needs to be stored to `establishedConnections` with this address. Because this address is guaranteed to be consistent with member list. Users configured address may not.

Reply:
I put back old behaviour.